product_docs/docs/pgd/5/routing/installing_proxy.mdx

```diff
@@ -53,9 +53,36 @@ You can set the log level for the PGD Proxy service using the top-level config p
 
 PGD Proxy uses endpoints given in the local config file only at proxy startup. After that, PGD Proxy retrieves the list of actual endpoints (route_dsn) from the PGD Proxy catalog. Therefore, the node option `route_dsn` must be set for each PGD Proxy node. See [route_dsn](../routing#configuration) for more information.
 
+##### Configuring health check
+
+PGD Proxy provides [HTTP(S) health check APIs](proxy#proxy-health-check). Add the below configuration parameters only if health check APIs are required. By default it is disabled.
+
+```yaml
+cluster:
+  name: cluster-name
+  endpoints:
+    - "host=bdr-a1 port=5432 dbname=bdrdb user=pgdproxy "
+    - "host=bdr-a3 port=5432 dbname=bdrdb user=pgdproxy "
+    - "host=bdr-a2 port=5432 dbname=bdrdb user=pgdproxy "
+  proxy:
+    name: "proxy-a1"
+    endpoint: "host=proxy-a1 port=6432 dbname=bdrdb user=pgdproxy "
+	  http:
```

Comment:
You appear to have some red marking in the code here which makes it through to the rendered version.

Reply:
Tabs, our mortal enemy! :)

Reply:
:)